services/app-web/src/components/ExpandableText/ExpandableText.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useLayoutEffect, useState } from 'react'
+import styles from './ExpandableText.module.scss'
+
+export type ExpandableTextProps = {
+    clamp: React.ReactElement | string
+    clampedLines?: string
+}
```

Comment:
I'm pretty sure the ability to pass in ReactElement for clamp will lead to strange things when passing in complicated elements. The reason for this was to add the bold text to Reason for unlock: and Changes made: text. There is probably a better way.